src/data/linux-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const linuxQuiz = [
+  {
+    Question:
+      "In Git, which command is used to set limits on file size?",
+    Answer: "ulimit",
+    Distractor1: "fsize",
+    Distractor2: "flimit -r",
+    Distractor3: "usize",
+    Explanation:
+      "ulimit is a builtin shell command used to show and set various restrictions on resource usage for a shell.",
```

Comment:
we want to avoid using direct quotes from external sites like these
https://wiki.linuxquestions.org/wiki/Ulimit

It is always best to write the explanations in your own word 👍 

```suggestion
      "ulimit is a Linux command that is used to get and set a limit on a file size.",
```